playwright.config.ts

```diff
@@ -22,7 +24,10 @@ export default defineConfig({
   /* Reporter to use. See https://playwright.dev/docs/test-reporters */
   reporter: 'html',
   /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
-  timeout: 1 * 60 * 1000,
+  timeout: TIMEOUT,
+  expect: {
+    timeout: TIMEOUT,
```

Comment:
```suggestion
    timeout: 12000,
```

Reply:
if we change the expect timeout to 2min, every single expect assertion can take up to 2 min.
changed it to 12s as i think it was what you were trying to achieve
let's change it and run CI tests to see

Reply:
I just reverted e2e stuff, since it's not related to this PR

Reply:
thank you